cmd/httprange_test.go

```diff
@@ -101,3 +101,37 @@ func TestHTTPRequestRangeSpec(t *testing.T) {
 		t.Errorf("Case %d: Expected errInvalidRange but: %v %v %d %d %v", i, rs, err1, o, l, err2)
 	}
 }
+
+func TestHTTPRequestRangeToHeader(t *testing.T) {
+	validRangeSpecs := []struct {
+		spec        string
+		errExpected bool
+	}{
+		{"bytes=0-", false},
+		{"bytes=1-", false},
+
+		{"bytes=0-9", false},
+		{"bytes=1-10", false},
+		{"bytes=1-1", false},
+		{"bytes=2-5", false},
+
+		{"bytes=-5", false},
+		{"bytes=-1", false},
+		{"bytes=-1000", false},
```

Comment:
let's add error cases as well @poornas such as

```
{"bytes=", true}
{"bytes=  ", true}
{"byte=", true}
{"bytes=A-B", true}
{"bytes=1-B", true}
{"bytes=B-1", true}
{"bytes=-1-1, true}
```

Reply:
these will be caught while parsing range spec. ToHeader() is on a valid range spec

Reply:
but we have to add these tests @poornas - I don't know how they will be caught